libraries/ESP8266WiFi/src/CertStoreBearSSL.h

```diff
@@ -0,0 +1,61 @@
+/*
+  CertStoreBearSSL.h - Library for Arduino ESP8266
+  Copyright (c) 2018 Earle F. Philhower, III
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _CERTSTORE_BEARSSL_H
+#define _CERTSTORE_BEARSSL_H
+
+#include <Arduino.h>
+#include <bearssl/bearssl.h>
+
+// Virtual base class for the certificate stores, which allow use
```

Comment:
If this is a base class for certificate stores, why is it specific to BearSSL? If we add a similar class for AxTLS, would it be derived from BearSSL one?

Reply:
hello:
have a stable version with this changes?